spec/factories/lettings_log.rb

```diff
@@ -23,15 +23,6 @@
       age1 { 17 }
       age2 { 19 }
     end
-    trait :conditional_section_complete do
-      tenancycode { Faker::Name.initials(number: 10) }
-      age1 { 34 }
-      sex1 { "M" }
-      ethnic { 2 }
-      national { 18 }
-      ecstat1 { 2 }
-      hhmemb { 1 }
-    end
```

Comment:
this was only being used in one place in a test I refactored, unless someone disagrees I don't find this trait name to describe something meaningful, but I might be missing some context for what conditional section means